documentation/source/user_section/tutorials/lesion-analysis/before-starting.rst

```diff
@@ -0,0 +1,14 @@
+Before starting this tutorial
+#############################
+
+#. Make sure that you have the following files in your working directory:
+
+   * ``single_subject/data/t2_lesion/t2.nii.gz``: A T2w anatomical image with fake lesion (because of the difficulty to share patient data).
+
+   You can get these files by downloading `data_lesion.zip <https://github.com/spinalcordtoolbox/sct_tutorial_data/archive/refs/heads/master.zip>`_.
```

Comment:
Ah! Before we merge, we will need to update this link to the new release for the tutorial data. (After performing steps 3 and 4.)

Reply:
If you're not already on it @valosekj, I have some time right now, so I'm going to create the release + update the link + merge the PR. :)

Reply:
Because of the recent rootlets commits, here is what I am going to do:

- Create a backup branch containing all of the rootlets commits
- Reset the master branch to contain only the lesion analysis commits
- Create the release for this PR (lesion only)
- Merge this PR
- Re-add the rootlets commits to the master branch 

On reflection, I wonder if it would be better to stick with PRs to the `sct_tutorial_data` repo after all. (We can still do the `master.zip` trick for testing PRs, but we would just replace `master` with the PR branch for `sct_tutorial_data`.)

Reply:
I have not done it yet. If you have some time, you can do it! Thank you, Joshua!

Agree with the solution for rootlets!